### ISSUE TITLE
Exception in 1.20.2

### ISSUE THREAD

**Issue by lerokko** [funkemunky/AntiVPN#46]
Whenever a player joins the following error message is printed in chat.
I am using `Paper-280 (MC: 1.20.2)` and I am not sure if functionality is affected since I do not have a VPN. (What it does do is fill my log.)

```
[Thu 22:13:39 WARN  KauriVPN] Task #1299594 for KauriVPN v1.9.1 generated an exception
java.lang.NullPointerException: Cannot invoke "java.net.InetSocketAddress.getAddress()" because the return value of "org.bukkit.entity.Player.getAddress()" is null
    at KauriVPN-1.9.1.jar//dev.brighten.antivpn.bukkit.BukkitListener$2.run(BukkitListener.java:121)
    at org.bukkit.craftbukkit.v1_20_R2.scheduler.CraftTask.run(CraftTask.java:101)
    at org.bukkit.craftbukkit.v1_20_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:480)
    at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1470)
    at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:446)
    at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1379)
    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1156)
    at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315)
    at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by funkemunky** [comment]
Released a new update 1.9.2, please let me know if this corrects it for you.

**Comment by lerokko** [comment]
I tested it right now in my staging environment. The exception is gone. It now looks just fine as it used to. Thank you for fixing this!
Tested on `Paper-318 (MC: 1.20.2)`
